src/ray/raylet/scheduling_resources.h

```diff
@@ -443,6 +443,11 @@ class ResourceIdSet {
   std::vector<flatbuffers::Offset<ray::protocol::ResourceIdSetInfo>> ToFlatbuf(
       flatbuffers::FlatBufferBuilder &fbb) const;
 
+  /// \brief Serialize this object as a string.
+  ///
+  /// \return A serialized string of this object.
+  const std::string Serialize() const;
```

Comment:
Add a TODO explaining that this can be removed when raylet client is switched to grpc?